nature.py

```diff
@@ -1,13 +1,52 @@
 import random
 from choices import Gender
-from defaults import Cost, Threshold
-from individual import Female, Male
+from defaults import Cost, Threshold, Ancestors
+from individual import Female, Male, Individual
+
+
+
 
 class Nature:
     def __init__(self, *args, **kwargs):
-        self.male_population = [Male(gender=Gender.MALE) for i in range(100)]
-        self.female_population = [Female(gender=Gender.FEMALE) for i in range(50)]
-    
+        # Define number of original ancestors
+        orig_male_ancestors_pop = int((Ancestors.MALE_PCT*Ancestors.ORIG_POP)/100)
```

Comment:
Percentage calculation is wrong. Should be:

int((Ancestors.MALE_PCT/Ancestors.ORIG_POP *100))

Reply:
It is correct: (x percentage of total) = (x * total)/100